ops/terraform/services/pipeline/main.tf

```diff
@@ -119,18 +120,154 @@ locals {
     }
     ccw = {
       enabled       = var.create_ccw_pipeline
+      name          = "bfd-${local.env}-${local.service}-ccw"
       instance_name = "ccw"
       instance_type = local.nonsensitive_ccw_service_config["instance_type"]
       tags = {
         Name = "bfd-${local.env}-${local.service}-ccw"
       }
     }
   }
-  pipeline_instances = { for k, v in local.pipeline_instance_configs : k => local.pipeline_instance_configs[k] if local.pipeline_instance_configs[k].enabled }
+  # FIXME: Replace with merged map with all variants when RDA is merged with CCW infrastructure
+  rda_pipeline_config = {
+    for k, v in local.pipeline_variant_configs : k => local.pipeline_variant_configs[k]
+    if local.pipeline_variant_configs[k].enabled && k == "rda"
+  }
+  # FIXME: Replace with merged map with all variants when RDA is merged with CCW infrastructure
+  ccw_pipeline_config = {
+    for k, v in local.pipeline_variant_configs : k => local.pipeline_variant_configs[k]
+    if local.pipeline_variant_configs[k].enabled && k == "ccw"
+  }
 }
 
+resource "aws_launch_template" "this" {
+  # FIXME: Use merged object instead of specific variant when RDA adopts ASG pattern
```

Comment:
Maybe we should have a ticket for this in the backlog so we remember to come back to this (and link it here)

Reply:
After speaking with Burling it's likely the RDA variant of the Pipeline will have a different mechanism altogether, so I've replaced these `FIXME`s scattered throughout `main.tf` with `TODO`s. I don't think we need to create an Issue for this in-particular since if we do happen to update RDA to use the same on-demand mechanisms as the CCW Pipeline consolidating the infrastructure will be a necessary part of that work.